apps/sleepphasealarm/app.js

```diff
@@ -38,13 +38,20 @@ function calc_ess(val) {
   }
 }
 
+// copied from alarm app
+// time in ms -> { hrs, mins }
+function decodeTime(t) {
```

Comment:
I have seen this function in alarm/app.js and
sched/sched.js. Should probably be moved to
library

Reply:
Thanks - yes, that's a good idea. Do you want to do a PR for it?

Reply:
I can try that later